src/Graphics/Vty/Input/Events.hs

```diff
@@ -43,6 +43,7 @@ data Event
     -- row. Some terminals report only that a button was released
     -- without specifying which one; in that case, Nothing is provided.
     -- Otherwise Just the button released is included in the event.
+    -- Mouse wheel release events are not reported.
```

Comment:
Actually, it might be better to just remove the comment; it's hard to talk about the nuance of *pressing* the scroll wheel because in some cases that might Just Work if the terminal emulator reports that as a middle button press. So saying we don't support it isn't even necessarily true!